experimental/dispatch_profiler/manifest.py

```diff
@@ -108,14 +157,18 @@ def append(self, dispatch_collection_list):
     for dispatch_collection in dispatch_collection_list:
       self.append_dispatch_collection(dispatch_collection)
 
+  def load(self):
+    """Loads the manifest with pre-defined dispatches for supported operations."""
+    matmul_dispatch_collection_list = MatmulGenerator(self.args).generate()
+    self.append(matmul_dispatch_collection_list)
+
+    #conv2d_dispatch_collection_list = Conv2dGenerator(self.args).generate()
+    #self.append(conv2d_dispatch_collection_list)
```

Comment:
I'm not sure those comments add much, I would consider removing them.

Reply:
I added those comments to show where and what changes will need to go in as we start working on conv or other dispatches, but we can remove these comments. I can help out ramp-up someone if someone gets interested in extending the tool to other dispatches, for conv2d most like i will be the one extending it :). So we can remove it.